### DISCUSSION TITLE
posts/travel-in-yunnan/

### DISCUSSION THREAD

**Discussion by giscus** [DriveFLY/MyBlog-comments#1] [Announcements]
# posts/travel-in-yunnan/

2023年的夏天，一段美好的时光

https://blog.drivefly.cn/posts/travel-in-yunnan/

**Comment by DriveFLY** [comment]
首评✌

**Comment by YamKH514** [comment]
Ciallo～(∠·ω< )⌒★

**Comment by Rising-Galaxy** [comment]
ヽ(✿ﾟ▽ﾟ)ノ   > biubiubiu～

**Comment by coldriver-chen** [comment]
大佬，你究竟是谁，我除了膜拜以外别无他法🧐
